.travis.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+CONF_CRYPTO=""
+CONF_OBJSTORE=""
+
+case $CRYPTO in
+botan)
+	CONF_CRYPTO="$CONF_CRYPTO --with-crypto-backend=botan --with-botan=/usr"
+	CONF_CRYPTO="$CONF_CRYPTO --disable-ecc --disable-gost"
```

Comment:
Too bad that Travis is using Ubuntu 12.04, which only include Botan 1.10.0. Thus not supporting ECC and GOST.